bidding_auction_services_aws_guide.md

```diff
@@ -18,11 +18,11 @@ The FLEDGE Bidding and Auction services will be open-sourced in Q2 2023. In addi
 
 
 #### Comparison to Google Cloud Platform (GCP)
-While this document focuses on AWS, a similar document will be published for GCP in Q2 2023. We intend to provide full GCP and AWS support for these services. Most implementation details will be, at a high level, similar. Two notable exceptions (ultimately with no impact on functionality) are:
+While this document focuses on AWS, a similar document was published for GCP in Q2 2023. We intend to provide full GCP and AWS support for these services. Most implementation details will be, at a high level, similar. A few notable exceptions (ultimately with no impact on functionality) are:
 
 1. GCP is based on a single [virtual machine instance][5], so a parent-child instance relationship will not be required to host the running trusted code in GCP.
 1. The [Envoy component][4] runs inside the Seller Frontend TEE in GCP.
-1. GCP uses a [service mesh][3] to handle its internal load balancing requirements and to allow message routing. AWS uses [load balancers][50]. 
+1. For B&A on AWS, each EC2 runs, outside the TEE: a health-checking script, and an instance of Envoy in a container. These support the [service mesh][3] for AWS. No such infrastructure is needed for mesh on GCP.
```

Comment:
Each EC2 instance in AWS runs the following outside TEE: a health-checking script, and an instance of Envoy in a container. These support the [service mesh][3] for AWS. No such infrastructure is needed for mesh on GCP.